packages/next/src/build/spinner.ts

```diff
@@ -33,9 +33,19 @@ export default function createSpinner(
     const origStopAndPersist = spinner.stopAndPersist.bind(spinner)
 
     const logHandle = (method: any, args: any[]) => {
-      origStop()
+      // Enter a new line before logging new message, to avoid
+      // the new message shows up right after the spinner in the saem line.
+      const isInProgress = spinner?.isSpinning
+      if (spinner && isInProgress) {
+        spinner.prefixText = '\r'
+        spinner.text = '\r'
```

Comment:
any reason this is `\r` instead of `\n`?

Reply:
It's for reseting the line, it will reset the current running spinner and move the cursor to the beginning of the line. I'll add a comment